packages/frontend/src/app/create-store/page.tsx

```diff
@@ -162,109 +171,102 @@ const StoreCreation = () => {
+    //Create manifest once KC is enrolled and client is connected + authenticated.
     if (
       relayClient &&
       shopId &&
       stateManager &&
       isStoreCreated &&
-      isConnected == Status.Complete
+      clientConnected == Status.Complete
     ) {
-      stateManager.manifest
-        .create(
-          {
-            name: storeName,
-            description,
-          },
-          shopId,
-        )
-        .catch((e) => {
-          debug(e);
-        });
-
-      // Create published and remove tags + update shopManifest and metadata
-      stateManager.tags
-        .create("visible")
-        .then((newPubId) => {
-          stateManager.tags
-            .create("remove")
-            .then()
-            .catch((e) => debug(e));
-
-          //Testing dom does not support FormData and test client will fail with:
-          //Content-Type isn't multipart/form-data
-          //so if it is a test env, we are skipping uploadBlob
-          let path = { url: "/" };
-          if (!process.env.TEST) {
-            relayClient!
-              .uploadBlob(avatar as FormData)
-              .then((res) => {
-                path = res;
-              })
-              .catch((e) => debug(e));
-          }
-
-          if (newPubId && path.url) {
-            stateManager.manifest
-              .update({
-                publishedTagId: newPubId.id,
-                setBaseCurrency: {
-                  tokenAddr: tokenAddr as `0x${string}`,
+      //Add address of kc wallet for all outgoing event verification.
+      const kc = localStorage.getItem("merchantKeyCard") as `0x${string}`;
+      const keyCardWallet = privateKeyToAccount(kc!);
+      stateManager.keycards.addAddress(keyCardWallet.address).then(() => {
+        stateManager.manifest
+          .create(
+            {
+              pricingCurrency: {
+                chainId: 1,
+                address: tokenAddr as `0x${string}`,
+              },
+              acceptedCurrencies: [
+                {
+                  address: tokenAddr as `0x${string}`,
                   chainId,
                 },
-                addAcceptedCurrencies: [
-                  {
-                    tokenAddr: tokenAddr as `0x${string}`,
-                    chainId,
-                  },
-                ],
-                addPayee: {
-                  addr: payeeAddr as `0x${string}`,
+              ],
+              payees: [
+                {
+                  address: payeeAddr as `0x${string}`,
                   callAsContract: false,
                   chainId,
                   name: "default",
                 },
-                profilePictureUrl: path.url,
+              ],
+              //TODO: UI for inputting shipping regions.
+              shippingRegions: [
+                {
+                  name: "test",
+                  country: "test country",
+                  postalCode: "test postal",
+                  city: "test city",
```

Comment:
```suggestion
                  name: "Default",
                  country: "",
                  postalCode: "",
                  city: "",
```

Empty string matches anything. Otherwise you can only ship to `test city` ;)